src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerDetailsController.js

```diff
@@ -0,0 +1,37 @@
+define(['angular', 'application/interviewer/tdprInterviewerModule'], function (angular, tdprInterviewerModule) {
+    tdprInterviewerModule.controller("tdprInterviewerDetailsController", function ($scope, $stateParams, tdprPersonService, $filter, $state, Notification, BandLevelEnum) {
+
+        $scope.BandLevelEnum = BandLevelEnum;
+
+        function init() {
+                $scope.person = tdprPersonService.getPerson();
```

Comment:
Why are you using tdprPersonService here instead of injecting person - that's why we're using resolve